src/components/layout/Cluster/Cluster.stories.tsx

```diff
@@ -70,6 +70,11 @@ WithGap.args = {
   gap: SpaceSizes.sm,
 };
 
+export const WithResponsiveGap = ClusterTemplate.bind({});
+WithResponsiveGap.args = {
+  gap: [SpaceSizes.sm, SpaceSizes.xxl],
+};
+
 export const WithHorizontalAlignment = ClusterTemplate.bind({});
 WithHorizontalAlignment.args = {
   ...WithGap.args,
```

Comment:
Please replace props on lines 101 and 102 with `as: 'ul'`. That will fix the Cluster As Navigation visual test